server/odc-service/src/main/java/com/oceanbase/odc/metadb/flow/UserTaskInstanceRepository.java

```diff
@@ -72,7 +72,8 @@ List<UserTaskInstanceEntity> findByResourceRoleIdentifierIn(
     @Transactional
     @Query("update UserTaskInstanceEntity as ut set ut.userTaskId=:#{#param.userTaskId},ut.status=:#{#param.status},"
             + "ut.approved=:#{#param.approved},ut.operatorId=:#{#param.operatorId},ut.comment=:#{#param.comment},"
-            + "ut.expireIntervalSeconds=:#{#param.expireIntervalSeconds} where ut.id=:#{#param.id}")
```

Comment:
Is it necessary to delete the update of `externalFlowInstanceId` here

Reply:
Emmm, on the contrary, there is adding the update of `externalFlowInstanceId`. The pursose is to update external approval at flow runtime.